tests/pkg_lora-serialization/tests-lora_serialization.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2018 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     unittests
+ * @{
+ *
+ * @file
+ * @brief       lora-serialization tests
+ *
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ * @author      Jose Ignacio Alamos <jose.alamos@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include <string.h>
+#include <stdio.h>
+#include "embUnit.h"
+#include "lora_serialization.h"
+
+#if defined(BOARD_NATIVE) && !defined(__clang__)
```

Comment:
> with cayenne-lpp we encountered some platform-dependent issues e.g. due to the way floats are handled on different platforms

@miri64 you mean this? I encountered problems with floats while running the test as well.